libs/core/langchain_core/_api/deprecation.py

```diff
@@ -208,7 +208,7 @@ def warn_if_direct_instance(
                 )
                 return cast(T, obj)
 
-        elif isinstance(obj, FieldInfo):
+        elif isinstance(obj, FieldInfoV1):
             from langchain_core.pydantic_v1 import Field
```

Comment:
```suggestion
```

Reply:
and below we can use FieldInfoV1?

and make a version for FieldInfoV2 so it works after 0.3 migration?